coding-challenges.js

```diff
@@ -11,6 +11,31 @@ var collections = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example2 (can be a different order): ["blue", "green", "pink"]
 // Expected output example3 (can be a different order): ["pink", "green"]
 
+const remove1 = (arr) => {
+    arr.shift()
+    let newArr = []
+    if (arr[0] === undefined){
+        return ("The array is empty")
+    }
+    for (let i=0; i<arr.length;i++) {
+        let random = Math.floor(Math.random()*arr.length)
+        let last = arr.slice(random, random+1)
+        better = last[0]
+        while (newArr.includes(better)){
+            random = Math.floor(Math.random()*arr.length)
+            last = arr.slice(random, random+1)
+            better = last[0]
+        }
+        newArr.push(better)
+    }
+
+
+    return newArr
+}
+
+console.log(remove1(collections));
+console.log(remove1(collections));
+console.log(remove1(collections));
```

Comment:
great! just make sure the run the function enough times to test all cases (i.e. condition when the array is empty)